ch-covidcertificate-backend-config/ch-covidcertificate-backend-config-wallet-ws/src/test/java/ch/admin/bag/covidcertificate/backend/config/wallet/ws/WalletControllerJsonTest.java

```diff
@@ -35,4 +44,39 @@ public void setup() {
         this.testHelper = new TestHelper(objectMapper);
         this.acceptMediaType = MediaType.APPLICATION_JSON;
     }
+
+    @Test
+    public void testTransferCodeValidity() throws Exception {
+        MockHttpServletResponse result =
+            mockMvc.perform(
+                    get(BASE_URL + "/config")
+                        .accept(acceptMediaType)
+                        .param("osversion", "ios12")
+                        .param("appversion", "ios-1.0.9")
```

Comment:
use last version before `2.7.0` (`2.6.1`)
```suggestion
                        .param("appversion", "ios-2.6.1")
```